build/common.ps1

```diff
@@ -160,7 +160,7 @@ Function Install-DotnetCLI {
     $vsMajorVersion = Get-VSMajorVersion
     $MSBuildExe = Get-MSBuildExe $vsMajorVersion
     $CliBranchListForTesting = & $msbuildExe $NuGetClientRoot\build\config.props /v:m /nologo /t:GetCliBranchForTesting
-    $CliBranchList = $CliBranchListForTesting.Split(';');
+    $CliBranchList = $CliBranchListForTesting[-1] -split ';'
```

Comment:
Is this the -1 a typo?

Reply:
Again, this is the fix

Reply:
If I do the following: 
```powershell

$CliBranchListForTesting = & MSBuild.exe build\config.props /v:m /nologo /t:GetCliBranchForTesting
$CliBranchList = $CliBranchListForTesting.Split(';');
$CliBranchList2 = $CliBranchListForTesting[-1] -split ';'

echo "Original: '$CliBranchList'"
echo "New: '$CliBranchList2'"
```

The output is
```
Original: '  master 5.0.100'
New: '0'
```

Reply:
fixed it, again!

Reply:
2nd attempt at fixing this. please check again.